### ISSUE TITLE
Extract font awesome definitions to separate file

### ISSUE THREAD

**Issue by joenas** [mcasimir/mobile-angular-ui#246]
Hi there and thanks for an awesome framework!

I have an issue when using this within a Rails app. Because of how Font Awesome is defined in [mobile-angular-ui-base.css](https://github.com/mcasimir/mobile-angular-ui/blob/master/dist/css/mobile-angular-ui-base.css) I get load errors. I have solved this by including the rails gem for font awesome. 
Would it be possible to extract the definitions into a separate file to make them optional?

Cheers /Jon

**Comment by RemcovandenBerg** [comment]
Can you please explain which files should look like what? Which branch or version of the files did you use? download dist / compiled sources / npm / generator?

And I am curious about the build process you are using with rails, I know nothing about that kind of stuff. So please explain a little bit more why this is causing your issue. Thanks.

**Comment by joenas** [comment]
Ah yes. 
I'm using the latest version through `bower`, so that would be the dist files. 
My suggestion would be to move [these](https://github.com/mcasimir/mobile-angular-ui/blob/master/dist/css/mobile-angular-ui-base.css#L1-L2379) lines to a separate file, for example `mobile-angular-ui-fonts` or `mobile-angular-ui-icons`, making it optional to include them. 

The main thing causing my issue is the [url](https://github.com/mcasimir/mobile-angular-ui/blob/master/dist/css/mobile-angular-ui-base.css#L202) for the fonts.
In Rails its not always trivial to set up assets with external gems, therefore I have to do stuff like this
```ruby
config.assets.paths << Rails.root.join("vendor","assets","bower_components","mobile-angular-ui","dist","fonts")
config.assets.precompile << %r(.*.(?:eot|svg|ttf|woff)$)
```
to precompile the assets in production. There is already a maintained gem for `font-awesome` for rails that does this for me. 
Did that make it any clearer? :confounded:

**Comment by RemcovandenBerg** [comment]
Yes a lot, thank you. I am not the owner of this project, so I have to discuss this first. 

But I think we should address that other people (including myself) aren't always using the gulp style of building applications. 

Perhaps we should split the distribution into a form where there is one splitted and componentized distro and another one buildable all-in-one gulp package.

**Comment by joenas** [comment]
> Perhaps we should split the distribution into a form where there is one splitted and componentized distro and another one buildable all-in-one gulp package.

That sounds lika a reasonable thing to do. I'm not very familiar with gulp but you could make all parts into components and perhaps include a file in dist like `mobile-angular-ui-full` that is built with all components.

**Comment by leolux** [comment]
I would like to use MAUI without using anything of fontawesome. Is that possible?
